### PR TITLE
19 token

### PR DESCRIPTION
**Pull request by seaparks**
@potocnyj there's a lot more that can be done here but I want to get out this change to make it optional before I forget.

### CODE REVIEW

File: alpine/README.md [hunk 1]
```diff
@@ -1,29 +1,30 @@
 # VividCortex Alpine Docker container
 
-Runs VividCortex agents in an [Alpine Linux](https://www.alpinelinux.org/) Docker container, for off-host monitoring (see
-[here](https://docs.vividcortex.com/getting-started/installing/#off-host-monitoring)).
+Runs VividCortex agents in an [Alpine Linux](https://www.alpinelinux.org/) Docker container, for off-host monitoring; see
+[here](https://docs.vividcortex.com/getting-started/off-host-installation/) for more detailis about off-host configuration requirements for your database. More general container installation documentation can be found [here](https://docs.vividcortex.com/getting-started/containerized-installation/).
 
 To use, you should build a image for your environment:
 
 	docker build --force-rm --build-arg VC_API_TOKEN=xxxxxxxxxxxxxxxx -t vcimage \
```
Comment (potocnyj) [inline alpine/README.md:8]: We probably need to change this, right? For reference, the commands at the top of the Dockerfile should be updated to mirror the README

Comment (seaparks) [reply alpine/README.md:8]: i see, you're saying i have to change this: https://github.com/VividCortex/docker/blob/19-token/alpine/Dockerfile#L6

Comment (potocnyj) [reply alpine/README.md:8]: Right, it should be consistent with the README

File: alpine/Dockerfile [hunk 2]
```diff
@@ -2,16 +2,20 @@ FROM alpine:3.8
...
 LABEL app=vividcortex
 
-# How to use:
-## docker build --force-rm --build-arg VC_API_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx -t vcimage https://raw.githubusercontent.com/VividCortex/docker/master/alpine/Dockerfile
-#opt1# default hostname : docker run --detach --interactive --tty --name=vividcortex vcimage
-#opt2# override hostname: docker run --env VC_HOSTNAME=mycontainer --detach --interactive --tty --name=vividcortex vcimage
-#opt3# monitor RDS      : docker run --env VC_HOSTNAME=mycontainer --env VC_DRV_MANUAL_HOST_URI=mysql://user:pass@domain.xyz:3306/db --detach --interactive --tty --name=vividcortex vcimage
+# How to build:
+# VC_API_TOKEN can be provided as --build-arg or as --env
+#opt1# build with token: docker build --force-rm --build-arg VC_API_TOKEN=xxxxx -t vcimage https://raw.githubusercontent.com/VividCortex/docker/master/alpine/Dockerfile
```
Comment (potocnyj) [inline alpine/Dockerfile:7]: This won't actually work with an `ENV` directive, you have to pass an arg and set the env value from it like this article outlines: https://vsupalov.com/docker-build-pass-environment-variables/

Comment (seaparks) [reply alpine/Dockerfile:7]: ohhhhhhhhhhhhhhh

**Pull request merged**